packages/google_maps_flutter/google_maps_flutter/pubspec.yaml

```diff
@@ -2,10 +2,7 @@ name: google_maps_flutter
 description: A Flutter plugin for integrating Google Maps in iOS and Android applications.
 repository: https://github.com/flutter/plugins/tree/main/packages/google_maps_flutter/google_maps_flutter
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+maps%22
-version: 2.1.10
-# Temporarily disable publishing to allow moving Android and iOS
-# implementations.
```

Comment:
This is gone on main now, so you just need to merge in the latest to fix this. Sorry about that!